tests/src/dir.common.props

```diff
@@ -30,10 +30,10 @@
 <!-- Setup the default output and intermediate paths -->
   <PropertyGroup>
     <BaseOutputPathWithConfig>$(ProjectDir)\..\binaries\tests\$(BuildOS).$(Platform).$(Configuration)\</BaseOutputPathWithConfig>
-    <BaseOutputPathWithConfig Condition="'$(__TestWorkingDir)' != ''">$(__TestWorkingDir)\</BaseOutputPathWithConfig>
+    <BaseOutputPathWithConfig Condition="'$(__TestBinDir)' != ''">$(__TestBinDir)\</BaseOutputPathWithConfig>
     <BinDir>$(BaseOutputPathWithConfig)\..</BinDir>
-    <BaseIntermediateOutputPath>$(BinDir)\intermediates\</BaseIntermediateOutputPath>
-    <__CMakeTestSlnDir Condition="'$(__CMakeTestSlnDir)' == ''">$(BaseOutputPathWithConfig)\CMake\</__CMakeTestSlnDir>
+    <BaseIntermediateOutputPath>$(ProjectDir)\..\binaries\tests\intermediates\$(BuildOS).$(Platform).$(Configuration)\</BaseIntermediateOutputPath>
```

Comment:
Too much verbosity,this is not necessary as right now the path will be like:
binaries\tests\intermediates\Common\Coreclr.TestWrapper\Windows_NT.x64.debug

and you are changing it to 
binaries\tests\intermediates\Windows_NT.x64.debug\Common\Coreclr.TestWrapper\Windows_NT.x64.debug